Ports/curl/package.sh

```diff
@@ -20,6 +20,8 @@ configure() {
     "${configopts[@]}" \
```

Comment:
Commit title grammar nitpick: "Let our CA certificate location **be** known to curl"

Reply:
Done